File: bookhub-front/src/dtos/employee/response/Employee-list.response.dto.ts
import { Status } from "@/apis/enums/StatusType";

export interface EmployeeListResponseDto {
  employeeId: number;
  employeeNumber: number;
  employeeName: string;
  branchName: string;
  positionName: string;
  authorityName: string;
  status: Status;
}

File: bookhub-back/src/main/java/com/example/bookhub_back/dto/employee/response/EmployeeListResponseDto.java
package com.example.bookhub_back.dto.employee.response;

import com.example.bookhub_back.common.enums.Status;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class EmployeeListResponseDto {
    private Long employeeId;
    private Long employeeNumber;
    private String employeeName;
    private String branchName;
    private String positionName;
    private String authorityName;
    private Status status;
}
